### ISSUE TITLE
Release Flatcar Container Linux Alpha 4081.0.0, Beta 4054.1.0 and Stable 3975.2.1

### ISSUE THREAD

**Issue by tormath1** [flatcar/Flatcar#1533]
The release of the following Flatcar Container Linux version is planning the week of the 2nd of September 2024

- Alpha 4081.0.0
- Beta 4054.1.0
- Stable 3975.2.1

## 1. Preparation

Owner: @tormath1 

- [x] A. Go/No-Go [hackmd doc](https://hackmd.io/team/flatcar) created from HackMD template for test tracking and release notes: https://hackmd.io/c1ySZhyZRLKsKP4aXY-wzw?view
- [x] B. All planned changes are merged & blocker bugs are fixed, last nightly reviewed for problems.
- [x] C. Check that the kernel PR are merged for the channel(s) to be released.

## 2. Build

Owner: @tormath1 

- [x] A. Branched off the `main` branch (only for new major).
- [x] B. Tagged all the build branches.
- [x] C. Started the Jenkins builds, post the links into this issue as comment.

### i. The Go/No-Go Meeting

Owner: @tormath1

- [x] A. Test status captured in the HackMD document.
- [x] B. All tests are Green.
- [x] C. Release notes / announcements added to Go/No Go doc from `container/image_changes` job by selecting `Timestamps: None`.
- [x] D. QA: Checked the diff for image file list, packages, and image size in the `container/image_changes` job output.
- [x] E. Conduct Matrix Go/No Go meeting (inc. release notes review).

### ii. Artifacts goes public

Owner: @tormath1 

- [x] A. Created the Release Notes on the `scripts` repo for the respective tags on GitHub as GitHub Releases.
- [x] B. Ran the `container/release` Release job.
- [x] C. Images uploaded with `copy-to-origin.sh`.
- [x] D. Symlink to "current" updated with `set-symlink.sh`.
- [x] E. Website updated with `./update-flatcar-versions.sh` and PR merged. https://github.com/flatcar/flatcar-website/pull/360

### iii. Payload

Owner: @tormath1 

- [x] A. Update payload signed/uploaded.
- [x] B. Release package published in Nebraska (Prerequisite: 3.ii.E should marked done).

### iv. Cloud Marketplace & Cluster API

Owner:

- A. Azure
    - [x] Offers updated and started publishing.
    - [x] Sent the offers to publish to Live.

- B. GCP
    - [x] Offer deployment package uploaded to Google Bucket & submit for verification.
    - [x] Offer Publish

- C. Cluster API for new Stable
  - [ ] Azure: Manually run `./azure-sig.sh` to create New Shared Community Gallery Images
  - [x] ~OpenStack: [documentation](https://gist.github.com/tormath1/acbae5c6cd12420bb8ea137e25655c99)~

## 4. Announcements

Owner: @tormath1 

- [x] A. Sent announcement to Mailing Lists
- [ ] B. Sent Tweet out to X (Twitter), and Toot out to Mastodon
- [x] C. Brief version announcement in slack (k8s slack #flatcar) and the Flatcar Matrix channel

**Comment by tormath1** [comment]
builds:
* alpha-4081.0.0: http://jenkins.infra.kinvolk.io:8080/job/container/job/sdk/1709/cldsv/
* beta-4054.1.0: http://jenkins.infra.kinvolk.io:8080/job/container/job/packages_all_arches/4612/cldsv/
* stable-3975.2.1: http://jenkins.infra.kinvolk.io:8080/job/container/job/packages_all_arches/4613/cldsv/

**Comment by t-lo** [comment]
GCE done (I think...) but the web interface got weird on me. This is likely https://github.com/flatcar/Flatcar/issues/1492 which will need some attention in the future. For now, the deployment manager marketplace updates still seem to work.

I tried the new "automated terraform conversion" mechanism the marketplace offers but it errored out on me. That new mechanism offers a list of release images to publish from buckets to marketplace, in which I selected all 4 latest images (alpha, beta, stable, LTS). However, when applying the changes it reported an error of "unable to replace existing images as replace property is not set".
(leaving this here as bread crumbs for later)
